mods/tnt/init.lua

```diff
@@ -538,7 +541,7 @@ function tnt.register_tnt(def)
 			description = def.description,
 			tiles = {tnt_top, tnt_bottom, tnt_side},
 			is_ground_content = false,
-			groups = {dig_immediate = 2, mesecon = 2, tnt = 1},
+			groups = {dig_immediate = 2, mesecon = 2, tnt = 1, flammable = 5},
```

Comment:
This is not the {{tnt:tnt_burning}} variant, this is the normal {{tnt:tnt}}, the burning variant is not flammable. The point of this is to make the normal tnt able to get started when there's an igniter next to it (thanks to {{on_burn}}).

But since I'm gonna change it to on_ignite, I'll leave that for another PR, as with the gunpowder.